### PR TITLE
Cyril bouchiat/renaming sec trace

### PR DESCRIPTION
**Pull request by cyrbouchiat**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/application_security/troubleshooting.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ If you experience unexpected behavior with Datadog Application Security Manageme
 
 ASM traces are rate-limited to 100 traces per second. Traces sent after the limit are not reported. Contact [Datadog support][1] if you need to change the limit.
 
-## No Suspicious Requests detected by ASM
+## No Security Traces detected by ASM
```
Comment (kayayarai) [inline content/en/security/application_security/troubleshooting.md:30]:
```suggestion
## No security traces detected by ASM
```

File: layouts/shortcodes/asm-protect.md [hunk 2]
```diff
@@ -5,10 +5,10 @@ ASM Protect goes beyond Threat Detection and enables you to take blocking action
 
 ASM leverages the same [tracing libraries][107] as Application Performance Monitoring (APM) to protect your applications against:
 
-- **Attacks**: ASM’s In-App WAF inspects all incoming traffic and uses pattern-matching to detect and block malicious traffic (suspicious requests).
+- **Attacks**: ASM’s In-App WAF inspects all incoming traffic and uses pattern-matching to detect and block malicious traffic (security traces).
```
Comment (kayayarai) [inline layouts/shortcodes/asm-protect.md:8]:
```suggestion
- **Attacks**: ASM's In-App WAF inspects all incoming traffic and uses pattern-matching to detect and block malicious traffic (security traces).
```

File: layouts/shortcodes/asm-protect.md [hunk 3]
```diff
@@ -5,10 +5,10 @@ ASM Protect goes beyond Threat Detection and enables you to take blocking action
...
 - **Attackers**: IP addresses and authenticated users that are launching attacks against your applications are detected from the insights collected by the libraries and flagged in Security Signals.
 
-Suspicious requests are blocked in real time by the Datadog tracing libraries. Blocks are saved in Datadog, automatically and securely fetched by the Datadog Agent, deployed in your infrastructure, and applied to your services. For details, read [How Remote Configuration Works][108]. 
+security traces are blocked in real time by the Datadog tracing libraries. Blocks are saved in Datadog, automatically and securely fetched by the Datadog Agent, deployed in your infrastructure, and applied to your services. For details, read [How Remote Configuration Works][108].
```
Comment (kayayarai) [inline layouts/shortcodes/asm-protect.md:11]:
```suggestion
Security traces are blocked in real time by the Datadog tracing libraries. Blocks are saved in Datadog, automatically and securely fetched by the Datadog Agent, deployed in your infrastructure, and applied to your services. For details, read [How Remote Configuration Works][108]. 
```

**Review by kayayarai — COMMENTED** [review]
Fixed a couple formatting nits (mostly that you didn't actually introduce). Otherwise looks fine. Although part of me wonders whether keeping "suspicious request" would be better in some instances -- it seems more human-understandable when talking about generic suspicious ongoings in your traffic, rather than calling it the name we give the feature. For example, in line 30 I think "When a suspicious request is detected" reads better than "When a security trace is detected". The change in the second part of the sentence seems fine. cc: @aliciascott

File: layouts/shortcodes/asm-protect.md (hunk 3 above)
Comment (kayayarai) [reply layouts/shortcodes/asm-protect.md:11]: Here, too, I think it should remain "Suspicious requests are blocked in real time"

**Comment by cyrbouchiat** [comment]
Thanks Kari for the review and suggestions.

I agree we can make a distinction between the general concept and the way
we call it in the product.
For consistency, we typically call the general concept "Security activity"
and the product naming "Security trace".
wdyt?

--

Cyril Bouchiat | Datadog
Product Manager, Application Security

See ASM in action <http://dtdg.co/asm-guided-tour>

3' ASM product demo  <https://www.youtube.com/watch?v=OaJHNuwaUmk>

File: static/images/security/application_security/Screenshot 2023-11-27 at 14.00.50.png
Comment (cyrbouchiat) [inline static/images/security/application_security/Screenshot 2023-11-27 at 14.00.50.png:1]: This change can be ignored. Sorry

**Comment by aliciascott** [comment]
Hi @cyrbouchiat just wanted to loop back on this and see if you were able to implement the changes suggested by @kayayarai and see if you had a target date for when you want this published? Thanks!

**Comment by cyrbouchiat** [comment]
> @cyrbouchiat
> 
> > we typically call the general concept "Security activity"
> > and the product naming "Security trace"
> 
> Hm, I think that's okay (better than calling both "security trace"). To me, "security activity" implies something done to improve security or some vaguely security-related activity, not something that violates or threatens security (as in a suspicious trace). But I also won't fight too hard about it :)

Part of the insights now logged as "security traces" are business-logic events that, by themselves are not threats or violations, but that can be used within detections. Think about a login event: its not suspicious. However, if this happen 100 times from the same user within a minute, this becomes a threat. "Security Activity" is a term we've been using at Sqreen that translates security-related information.

**Comment by kayayarai** [comment]
> Part of the insights now logged as "security traces" are business-logic events that, by themselves are not threats or violations, but that can be used within detections. Think about a login event: its not suspicious. However, if this happen 100 times from the same user within a minute, this becomes a threat. "Security Activity" is a term we've been using at Sqreen that translates security-related information.

Ah, ok, that is clarifying yes.

File: layouts/shortcodes/asm-protect.md (hunk 3 above)
Comment (kayayarai) [reply layouts/shortcodes/asm-protect.md:11]:
```suggestion
Security activity is blocked in real time by the Datadog tracing libraries. Blocks are saved in Datadog, automatically and securely fetched by the Datadog Agent, deployed in your infrastructure, and applied to your services. For details, read [How Remote Configuration Works][108]. 
```

Comment (kayayarai) [reply layouts/shortcodes/asm-protect.md:11]: ok, let's just capitalize the first word in line 11, and leave the terminology as-is

**Review by kayayarai — APPROVED** [review]

**Pull request merged**